docs/components/nav_drawer.md

```diff
@@ -0,0 +1,324 @@
+<!--docs:
+title: "Material <component>"
+layout: detail
+section: components
+excerpt: "Navigation drawers provide access to destinations in your app."
+iconId:
+path: /catalog/NavigationDrawers/
+-->
+
+# Navigation drawer
+
+A [navigation drawer](https://material.io/components/navigation-drawer) provides access to destinations in your app.
+
+![Hero navigation drawer image](assets/nav_drawer/NavDrawer-hero.png)
+
+## Contents
+
+* [Using a navigation drawer](#using-a-navigation-drawer)
+* [Making navigation drawers accessible](#making-navigation-drawers-accessible)
```

Comment:
Only have top level items in the contents

Reply:
Done